packages/flutter/lib/src/widgets/container.dart

```diff
@@ -365,77 +410,43 @@ class Container extends StatelessWidget {
   /// method throws an [UnsupportedError].)
   final Clip clipBehavior;
 
-  EdgeInsetsGeometry? get _paddingIncludingDecoration {
-    if (decoration == null) {
-      return padding;
-    }
-    final EdgeInsetsGeometry decorationPadding = decoration!.padding;
-    if (padding == null) {
-      return decorationPadding;
-    }
-    return padding!.add(decorationPadding);
-  }
-
-  @override
-  Widget build(BuildContext context) {
-    Widget? current = child;
+  EdgeInsetsGeometry? get _paddingIncludingDecoration => switch ((decoration?.padding, padding)) {
+        (null, final pad) || (final pad, null) => pad,
```

Comment:
Here and elsewhere please follow our style guide when naming things: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-abbreviations